### DISCUSSION TITLE
v2.10.2's internal software PID Duty Clamp Min

### DISCUSSION THREAD

**Discussion by Terracotta-6** [artisan-roaster-scope/artisan#1482] [General]
![679643c648ce37314788d7fa3589f89](https://github.com/artisan-roaster-scope/artisan/assets/149906183/e2167fb1-4aed-40f6-8e80-02b87757288a)

**Comment by Terracotta-6** [comment]
![da33f18236bf5f8f9bec40465038df5](https://github.com/artisan-roaster-scope/artisan/assets/149906183/22da3f35-f053-4f2c-ae1d-5a646e57be4b)

**Comment by MAKOMO** [comment]
Thanks for reporting. This was an issue with the SV/Duty device reporting the wrong data, not respecting the duty limits correctly. Just fixed this.

**Comment by Terracotta-6** [comment]
![e20693339832897d29c5ccede3ca043](https://github.com/artisan-roaster-scope/artisan/assets/149906183/2643f82f-6f2c-44a6-b855-b0035581aa21)

**Comment by Terracotta-6** [comment]
https://github.com/artisan-roaster-scope/artisan/assets/149906183/5690e6bd-834e-48fd-841c-3729ae57f525

**Reply by Terracotta-6** [reply]
has been fixed.

**Comment by Terracotta-6** [comment]
https://github.com/artisan-roaster-scope/artisan/assets/149906183/b309b35c-d3da-4df3-afe7-81218c481c41


https://github.com/artisan-roaster-scope/artisan/assets/149906183/c9a8f981-4dab-436d-b95f-6ce4f60a059c

**Comment by Terracotta-6** [comment]
has been fixed.
